### PR TITLE
Mount a volume to save the logs to the VM

### PR DESCRIPTION
**Pull request by FredrikFolkesson**
We are now mounting a volume on the elasticsearch container to the VM node so the logs should be saved until the VMs are removed.

the folder on the VM is `/esdatadir`

created like this:

```
mkdir esdatadir
chmod g+rwx esdatadir
chgrp 1000 esdatadir
```

If the host does not have this folder E.G you spinning up the example on your machine(s) the folder will be created.

### CODE REVIEW

File: docker-compose.logging.yml [hunk 1]
```diff
@@ -17,6 +17,8 @@ services:
     # This service only runs as one instance in our stack and is
     # used by Kibana (consuming) and Logstash (storing).
     image: docker.elastic.co/elasticsearch/elasticsearch-oss:6.1.1
+    volumes:
+      - /esdatadir:/usr/share/elasticsearch/data:rw
```
Comment (peol) [inline docker-compose.logging.yml]: AFAIK you cannot mount host volumes like this in a Swarm environment?

File: .circleci/config.yml [hunk 2]
```diff
@@ -100,7 +100,7 @@ jobs:
               export DOCKER_MACHINE_NAME=${DOCKER_AWS_MANAGER_NAME}
               scp -o StrictHostKeyChecking=no ubuntu@${DOCKER_AWS_MANAGER_IP}:/home/ubuntu/DEPLOY_SSL_* ~/qliktive-custom-analytics/secrets/
               cd ..
-              ACCEPT_EULA=yes AUTH_STRATEGY="github" docker-compose -f docker-compose.yml -f docker-compose.pregen-ssl.yml config > docker-compose.merged.yml
+              ACCEPT_EULA=yes AUTH_STRATEGY="github" docker-compose -f docker-compose.yml -f dicjer.cinoise,circleci.yml -f docker-compose.pregen-ssl.yml config > docker-compose.merged.yml
```
Comment (peol) [inline .circleci/config.yml]: Hehe, you might want to re-check this one

File: docker-compose.circleci.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+version: "3.3"
```
Comment (peol) [inline docker-compose.circleci.yml]: Rename file to something descriptive like `docker-compose.persistence.yml`?

Comment (FredrikFolkesson) [reply docker-compose.circleci.yml]: I am thinking we might add other specific stuff here for our deployment in the future.

Comment (peol) [reply docker-compose.circleci.yml]: Fair enough

**Pull request merged**